include/experimental/__p1673_bits/blas1_linalg_swap.hpp

```diff
@@ -100,13 +101,14 @@ void linalg_swap(inout_object_1_t v1,
 template<class ExecutionPolicy,
          class inout_object_1_t,
          class inout_object_2_t>
-void linalg_swap(ExecutionPolicy&& /* exec */,
-                 inout_object_1_t v1,
-                 inout_object_2_t v2)
+void swap_elements(ExecutionPolicy&& /* exec */,
+                   inout_object_1_t v1,
+                   inout_object_2_t v2)
 {
-  linalg_swap(v1, v2);
+  swap(v1, v2);
```

Comment:
Should this perhaps be `swap_elements(v1, v2)` instead?  `swap(v1, v2)` would swap the `mdspan` objects instead of their elements.

Reply:
This should be fixed now. Thanks for catching it!